src/sass/components/_header.scss

```diff
@@ -160,63 +161,128 @@ header {
       }
     }
 
-    .search-form {
-      border: 1px solid $form-border-color;
-      display: inline-flex;
-      height: 45px;
-      color: $text-color;
+    .mobile-menu {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
 
-      .category {
-        border-right: 1px solid $form-border-color;
-        display: flex;
-        align-items: center;
-        position: relative;
+      .search-form {
+        border: 1px solid $form-border-color;
+        display: inline-flex;
+        height: 45px;
+        color: $text-color;
 
-        i:first-child {
-          color: $primary;
-          position: absolute;
-          left: 10px;
-          z-index: 0;
-        }
+        .category {
+          border-right: 1px solid $form-border-color;
+          display: flex;
+          align-items: center;
+          position: relative;
 
-        i:last-child {
-          position: absolute;
-          right: 10px;
-          z-index: 0;
+          i:first-child {
```

Comment:
Obie reguły posiadają względnie podobne własności, może da się to jakoś połączyć?

Reply:
chodzi oczywiście o ten `:first-child` i `last-child`